website/src/components/Submission/SubmissionPageWrapper.astro

```diff
@@ -6,10 +6,11 @@ import NeedToLogin from '../common/NeedToLogin.astro';
 
 interface Props {
     title: string;
+    disablePageTitle?: boolean;
```

Comment:
nbd but I guess this may not be needed any more? But it's fine if you want to keep it anyway

Reply:
I know it's a bit of a hack - IMO the header shouldn't be in the wrapper, every page should have it's own wrapper.

It's still needed at least if we want the layout to stay the way it is. That is - have the header line up with the form. The header in the wrapper is left-justified, and I think that doesn't look very nice, so I'd keep it for now.

Reply:
Thanks for explaining!